src/hec/firo/SqLiteEnsemble.java

```diff
@@ -0,0 +1,177 @@
+package hec.firo;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+/// <summary>
+  /// Reads/Writes Ensemble data to SQL tables
+  /// each ensemble member is written to a blob
+  /// with optional compressions
+  /// </summary>
+  public class SqLiteEnsemble
```

Comment:
I believe this is in response to my issue - but I am not sure. it is still in the namespace hec.firo.

I think an abstraction would be useful here
EnsembleReader{
public string getFilePath();
public Ensemble Read(arguments);
public bool Write(Ensemble);
public bool Write(Ensemble[]);
}

Reply:
SqLiteEnsemble.java does not exist in the branches.